packages/starknet-snap/src/utils/constants.ts

```diff
@@ -33,7 +38,7 @@ export const STARKNET_MAINNET_NETWORK: Network = {
   name: 'Starknet Mainnet',
   chainId: constants.StarknetChainId.SN_MAIN,
   baseUrl: 'https://alpha-mainnet.starknet.io',
-  nodeUrl: 'https://starknet-mainnet.infura.io/v3/60c7253fb48147658095fe0460ac9ee9',
```

Comment:
Why removing nodeUrl ?

Reply:
it is no longer use, and try not keep those id in the source code